resources/js/requests/components/FilterMobile.vue

```diff
@@ -0,0 +1,281 @@
+            <i class="fas fa-caret-down"/>
+          </button>
+          <div
+            class="dropdown-menu"
+            aria-labelledby="statusDropdown"
+          >
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption('In Progress', 'status', 'fas fa-circle text-warning')"
+            >
+              <i class="fas fa-circle text-warning"/>
+              {{ $t('In Progress') }}
+            </a>
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption('Completed', 'status', 'fas fa-circle text-primary')"
+            >
+              <i class="fas fa-circle text-primary"/>
+              {{ $t('Completed') }}
+            </a>
+          </div>
+        </div>
+      </b-col>
+      <b-col cols="1" class="d-flex">
+        <div
+          v-if="showDropdowns && type === 'requests'"
+          class="dropdown"
+        >
+          <button
+            id="requestsDropdown"
+            class="btn btn-secondary dropdown-toggle"
+            type="button"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <i :class="selectedIconFilter"/>
+            <i class="fas fa-caret-down"/>
+          </button>
+          <div
+            class="dropdown-menu"
+            aria-labelledby="requestsDropdown"
+          >
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption(`(requester = 'admin')`, 'filter', 'fas fa-user')"
+            >
+              <i class="fas fa-user"/>
+              {{ $t('Requested by Me') }}
+              <i v-if="selectedIconFilter=== 'fas fa-user'" class="fas fa-check ml-auto text-success"/>
+            </a>
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption(`(participant = 'admin')`, 'filter', 'fas fa-users')"
+            >
+              <i class="fas fa-users"/>
+              {{ $t('With me as Participant') }}
+              <i v-if="selectedIconFilter === 'fas fa-users'" class="fas fa-check ml-auto text-success"/>
+            </a>
+          </div>
+        </div>
+        <div
+          class="dropdown"
+          v-if="showDropdowns && type === 'tasks'"
+        >
+          <button
+            id="tasksDropdown"
+            class="btn btn-secondary dropdown-toggle"
+            type="button"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <i class="fas fa-list"/>
+            <i class="fas fa-caret-down"/>
+          </button>
+          <div
+            class="dropdown-menu"
+            aria-labelledby="tasksDropdown"
+          >
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption('By Due Date', 'orderBy')"
```

Comment:
Review if this work with translations, looks like we are using a translatable label to select an option

Reply:
not necessary in this section is data to send an API